distribution/docs/src/main/resources/content/_sources/confluence-federated-source.adoc

```diff
@@ -1,14 +1,15 @@
 :title: Federated Source for Atlassian Confluence(R)
 :type: source
 :status: published
-:link: _federated_source_for_atlassian_confluence
+:link: _federated_source_for_atlassian_confluence_r
```

Comment:
is the `_r` at the end intentional?

Reply:
Yes. The `(R)` in the title gets rendered that way. Otherwise the link breaks